core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/deploysemantics/PhysicalOp.scala

```diff
@@ -181,8 +181,6 @@ case class PhysicalOp(
       Map.empty,
     outputPorts: Map[PortIdentity, (OutputPort, List[PhysicalLink], Either[Throwable, Schema])] =
       Map.empty,
-    // input ports that are blocking
-    blockingInputs: List[PortIdentity] = List(),
     // output ports that are blocking
     blockingOutputs: List[PortIdentity] = List(),
```

Comment:
can you make this a property of `OutputPort` instead of a property of a `PhysicalOperator`?

Reply:
done